sim-administration/simmanager/src/main/kotlin/org/ostelco/simcards/inventory/SimInventoryDAO.kt

```diff
@@ -211,7 +214,7 @@ class SimInventoryDAO(private val db: SimInventoryDBWrapperImpl) : SimInventoryD
     //
 
     override fun insertAll(entries: Iterator<SimEntry>): Either<SimManagerError, Unit> =
-        db.insertAll(entries)
+            db.insertAll(entries)
```

Comment:
A TAB? Prefer space myself - except in Makefiles

Reply:
*blush*